### DISCUSSION TITLE
RayGUI cheatsheet when?

### DISCUSSION THREAD

**Discussion by mlepe** [raysan5/raylib#4045] [Features]
Hey!
I'm wondering if you're planning on releasing documentation, or at least a cheatsheet, for RayGUI.
I'm an amateur at C programming at best, and using the visual builder makes it hard for me to see the connections with the actual code with that makes sense.
Anyways, good job @raysan5!

**Comment by mamadof** [comment]
yes, i like raygui too and love to see a cheat sheet like raylib has.
love and respect @raysan5

**Comment by raysan5** [comment]
No plans for it, no resources to create and maintain it available, just check the `raygui.h` header.

**Comment by Ali-Ogli** [comment]
Well, it's 2025 and I didn't see anything like that. If you found anything helpful please tell because raygui with all it's examples aren't good source for me to learn what it has.

**Comment by Desire4444** [comment]
It's 2026 at this point and also wondering if maybe anyone has come across a cheat sheet  please.
